src/test/integration/tests/Deposit_Queue_Complete.t.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "src/test/integration/User.t.sol";
+import "src/test/integration/IntegrationChecks.t.sol";
+
+contract Integration_Deposit_QueueWithdrawal_Complete is IntegrationCheckUtils {
+
+    /// Randomly generates a user with different held assets. Then:
+    /// 1. deposit into strategy
+    /// 2. queueWithdrawal
+    /// 3. completeQueuedWithdrawal"
+     function testFuzz_deposit_queueWithdrawal_completeAsTokens(uint24 _random) public {
+        // Configure the random parameters for the test
+        _configRand({
+            _randomSeed: _random,
+            _assetTypes: HOLDS_LST | HOLDS_ETH | HOLDS_ALL,
+            _userTypes: DEFAULT | ALT_METHODS
+        });
+
+        // Create a staker with a nonzero balance and corresponding strategies
+        (User staker, IStrategy[] memory strategies, uint[] memory tokenBalances) = _newRandomStaker();
+
+        // 1. Deposit into strategy
+        staker.depositIntoEigenlayer(strategies, tokenBalances);
+        uint[] memory shares = _calculateExpectedShares(strategies, tokenBalances);
+        check_Deposit_State(staker, strategies, shares);
+
+        // Ensure staker is not delegated to anyone post deposit
+        assertFalse(delegationManager.isDelegated(address(staker)), "Staker should not be delegated after deposit");
+
+        // 2. Queue Withdrawal
+        IDelegationManager.Withdrawal[] memory withdrawals = staker.queueWithdrawals(strategies, shares);
+
+        // 3. Complete Queued Withdrawal
+        cheats.roll(block.number + delegationManager.withdrawalDelayBlocks());
+        for (uint i = 0; i < withdrawals.length; i++) {
+            uint[] memory expectedTokens = _calculateExpectedTokens(strategies, shares);
+            IERC20[] memory tokens = staker.completeWithdrawalAsTokens(withdrawals[i]);
+            check_Withdrawal_AsTokens_State(staker, address(0), withdrawals[i], strategies, shares, tokens, expectedTokens);
```

Comment:
I added address(0) for now, so the file won't compile. Can you overload the `check_WithdrawalAstokens_State` to accept a non-user for the operator so that these tests will pass?